colour/utilities/common.py

```diff
@@ -302,3 +305,48 @@ def is_integer(x):
     """
 
     return abs(x - round(x)) <= INTEGER_THRESHOLD
+
+
+def filter_kwargs(function, **kwargs):
```

Comment:
Is this a good solution to a specific otherwise unsolvable problem?
Seems like this might eat otherwise potentially useful errors.